utils/modularize.js

```diff
@@ -186,9 +194,11 @@ function convert( path, exampleDependencies, ignoreList ) {
 		.sort()
 		.toString();
 
+	var imports = '';
+
 	// core imports
 
-	var imports = `import {${keys}\n} from "../../../build/three.module.js";`;
+	if ( keys ) imports += `import {${keys}\n} from "../../../build/three.module.js";`;
```

Comment:
This change is necessary since example files can have now dependency to the core but to other example files (see `RenderPass`).